osu.Game/Screens/Edit/Screens/Setup/Components/LabelledComponents/LabelledSliderBar.cs

```diff
@@ -0,0 +1,251 @@
+            }
+        }
+
+        private float sliderNormalPrecision = 1;
+        public float SliderNormalPrecision
+        {
+            get => sliderNormalPrecision;
+            set
+            {
+                sliderNormalPrecision = value;
+                sliderBar.NormalPrecision = value;
+            }
+        }
+
+        private float sliderAlternatePrecision = 1;
+        public float SliderAlternatePrecision
+        {
+            get => sliderAlternatePrecision;
+            set
+            {
+                sliderAlternatePrecision = value;
+                sliderBar.AlternatePrecision = value;
+            }
+        }
+
+        private string tooltipTextSuffix = "";
+        public string TooltipTextSuffix
+        {
+            get => tooltipTextSuffix;
+            set
+            {
+                tooltipTextSuffix = value;
+                sliderBar.TooltipTextSuffix = value;
+            }
+        }
+
+        public string LabelText
+        {
+            get => label.Text;
+            set => label.Text = value;
+        }
+
+        public string BottomLabelText
+        {
+            get => bottomText.Text;
+            set
+            {
+                bottomText.Text = value;
+                Height = normal_height + (value != "" ? 20 : 0);
+            }
+        }
+
+        public string LeftTickCaption
+        {
+            get => sliderBar.LeftTickCaption;
+            set => sliderBar.LeftTickCaption = value;
+        }
+        public string MiddleTickCaption
+        {
+            get => sliderBar.MiddleTickCaption;
+            set => sliderBar.MiddleTickCaption = value;
+        }
+        public string RightTickCaption
+        {
+            get => sliderBar.RightTickCaption;
+            set => sliderBar.RightTickCaption = value;
+        }
+
+        public float LabelTextSize
+        {
+            get => label.TextSize;
+            set => label.TextSize = value;
+        }
+
+        public MarginPadding Padding
+        {
+            get => base.Padding;
+            set => base.Padding = value;
+        }
+
+        public MarginPadding LabelPadding
+        {
+            get => label.Padding;
+            set => label.Padding = value;
+        }
+
+        public MarginPadding SliderBarPadding
+        {
+            get => sliderBar.Padding;
+            set => sliderBar.Padding = value;
+        }
+
+        public Color4 LabelTextColour
+        {
+            get => label.Colour;
+            set => label.Colour = value;
+        }
+
+        public Color4 BackgroundColour
+        {
+            get => box.Colour;
+            set => box.Colour = value;
+        }
+
+        public LabelledSliderBar()
+        {
+            Masking = true;
+            CornerRadius = corner_radius;
+            RelativeSizeAxes = Axes.X;
+            Height = normal_height;
+
+            InternalChildren = new Drawable[]
+            {
+                new Container
```

Comment:
This container doesn't look like it does anything - corner radius/masking is set on the `LabelledSliderBar`